src/ui/iconloader.h

```diff
@@ -22,13 +22,21 @@
 
 class IconLoader {
  public:
+  enum IconType {
+    base = 0,
```

Comment:
Ok, i will try to use enum class. did you mean to change base to Base, provider to Provider and so on ?